src/System.Configuration.ConfigurationManager/src/Resources/System.Configuration.ConfigurationManager.rd.xml

```diff
@@ -2,7 +2,8 @@
 <Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
   <Library Name="*System.Configuration.ConfigurationManager*">
     <Assembly Name="System.Configuration.ConfigurationManager">
-       <Type Name="System.Configuration.ClientConfigurationHost" Dynamic="Required All" Activate="Required All" />
+	<Type Name="System.Configuration.ClientConfigurationHost" Dynamic="Required All" />
```

Comment:
It'd be good to have a short comment on each Type element explaining why that type needs to be rooted.